app/templates/editor/level/modal/artisan-guide-modal.jade

```diff
@@ -17,7 +17,7 @@ block modal-body-content
       a(href='https://github.com/codecombat/codecombat/wiki/Artisan-How-To-Index', target='_blank') Basic How-tos
     div
       a(href='http://direct.codecombat.com/community', target='_blank') Artisan Rankings
```

Comment:
Shouldn't be linking to direct.codecombat.com. That's only for testing the site w/out the cache.